lib/iris/tests/unit/fileformats/pp/test_PPField.py

```diff
@@ -43,7 +43,7 @@
 ]
 
 
-class TestPPField(PPField):
+class DummyPPField(PPField):
```

Comment:
```
  /tmp/cirrus-ci-build/lib/iris/tests/unit/fileformats/pp/test_PPField.py:46: PytestCollectionWarning: cannot collect test class 'TestPPField' because it has a __init__ constructor (from: lib/iris/tests/unit/fileformats/pp/test_PPField.py)
```
This one doesn't seem to be a test class, so we can call it something else.